alcs-frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "@ng-select/ng-option-highlight": "^12.0.6",
     "@types/validator": "^13.15.2",
     "angular-mentions": "^1.5.0",
+    "jsdom": "^26.1.0",
```

Comment:
Why are we adding a new package? Is this a dependency of another package where we want to change the version?